components/automate-ui/src/app/helpers/auth/authorized.ts

```diff
@@ -60,14 +62,28 @@ export class AuthorizedChecker {
       this.store.dispatch(new GetUserParamPerms(payload)));
   }
 
+  private inflate(inputs: CheckObj[]): CheckObj[] {
+    inputs.forEach(check => {
+      if (this.placeholderRE.test(check.endpoint)) {
+        let newEndpoint = check.endpoint;
+        const list = isArray(check.paramList) ? check.paramList : [check.paramList];
+        list.forEach(val => newEndpoint = newEndpoint.replace(this.placeholderRE, val));
+        check.inflatedEndpoint = newEndpoint;
+      }
+    });
+    return inputs;
+  }
+
   destroy() {
     this.isDestroyed.next(true);
     this.isDestroyed.complete();
   }
 
+  // For a parameterized endpoint, we need to check the inflated endpoint--
+  // with parameters filled in--so we know permissions for the endpoint with particular values.
   private permsPopulated(perms: IndexedEntities<UserPermEntity>): boolean {
     return every(
-      (check: CheckObj) => has(check.endpoint, perms),
+      (check: CheckObj) => has(check.inflatedEndpoint || check.endpoint, perms),
```

Comment:
inflatedEndpoint will be empty for non-parameterized endpoints so we are then checking against the original endpoints, as always. But crucially we are now correctly checking parameterized endpoints.